litani

```diff
@@ -537,13 +554,32 @@ def init(args):
     except KeyError:
         run_id = str(uuid.uuid4())
 
-    cache_dir = pathlib.Path(
-        tempfile.gettempdir()) / "litani" / "runs" / run_id
-    cache_dir.mkdir(parents=True)
+    if args.output_prefix:
+        output_prefix = pathlib.Path(args.output_prefix).resolve()
+    else:
+        output_prefix = pathlib.Path(tempfile.gettempdir())
+
+    if args.output_directory:
+        cache_dir = pathlib.Path(args.output_directory).resolve()
+    else:
+        cache_dir = output_prefix / "litani" / "runs" / run_id
+
+    try:
+        cache_dir.mkdir(parents=True)
+    except FileExistsError as err:
+        logging.error("Output directory already exists")
+        logging.error(str(err))
```

Comment:
I don't particularly like printing stringified exceptions for this sort of user-facing tool, I'm not sure that it adds much value and it's more output for users to read/get confused by. I'd prefer just `logging.error("Output directory '%s' already exists", cache_dir)`.